cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
Yup. I think properties_form should be simply a property of cloud.Provider that will resolve the prop_region defined in cloud module.